packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer-card.jsx

```diff
@@ -1,11 +1,9 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { DesktopWrapper, MobileWrapper, Collapsible } from '@deriv/components';
-import ContractCard from 'App/Components/Elements/ContractCard';
-import CardBody from './contract-drawer-card-body.jsx';
-import CardFooter from './contract-drawer-card-footer.jsx';
-import CardHeader from './contract-drawer-card-header.jsx';
+import { DesktopWrapper, MobileWrapper, Collapsible, ContractCard } from '@deriv/components';
 import { SwipeableContractDrawer } from './swipeable-components.jsx';
+import CardFooter from './contract-drawer-card-footer.jsx';
+import { card_labels, getContractTypeDisplay } from '../../../../Constants/contract';
```

Comment:
Can we use alias path instead of relative?

Reply:
Updated.